### PR TITLE
feat(git):[TRI-281] Add commit-message git-hook

### PR DESCRIPTION
**Pull request by ds-jhartmann**
Added git-hook for commit messages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,5 +65,22 @@ If you want to run it against connector you need to add the following VM options
...
+
+Detailed pattern can be found here: [commit-msg](dev/commit-msg)
+
+###Installation
+```shell
+curl --fail -o .git/hooks/commit-msg https://raw.githubusercontent.com/catenax/ItemRelationshipServices/main/dev/commit-msg \
+  && chmod 500 .git/hooks/commit-msg
+```
+
+For further information please see https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commit-message-format
```
Comment (mkanal) [inline README.md:83]: What information should be get from an angular.js site. Probably we should refer another page which is desrcibing the semantic commit messages. Angular.js is more frontend and could irritate  the developer

File: dev/commit-msg [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+if [ -z "$1" ]; then
+	echo "Missing argument (commit message). Did you try to run this manually?"
```
Comment (mkanal) [inline dev/commit-msg:4]: What does "Did you try to run this manually" mean in this context. Please give more context in the message.

File: dev/commit-msg [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+# ignore merge requests
+if echo "$commitTitle" | grep -qE "^Merge branch '.*"; then
+	echo "Commit hook: ignoring branch merge"
+	exit 0
+fi
+
+# check semantic versioning scheme
+if ! echo "$commitTitle" | grep -qE '^(build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test)\([a-z0-9/-]+\)\:\[[A-Z0-9-]+\].*$'; then
+	echo "Your commit title did not follow semantic versioning: $commitTitle"
+	echo "Please see https://github.com/catenax/ItemRelationshipServices#commit-messages"
```
Comment (mkanal) [inline dev/commit-msg:19]: is it possible to use a a var instead of a url. If we change the repository we have to change the url .

Comment (mkanal) [reply dev/commit-msg:19]: ;-) Thank you very much.

**Review by mkanal — CHANGES_REQUESTED** [review]
Hi Jaro, 
Review was finished. 
Thank you very much,

File: dev/commit-msg (hunk 2 above)
Comment (ds-jhartmann) [reply dev/commit-msg:4]: This message will be displayed when you don't provide any commit message or execute the script on your local machine without arguments like this: "./commit-msg".
This was already present in the example and I didn't see a reason to remove it.
IMO we don't need to specify this further.

**Review by mkanal — APPROVED** [review]

**Pull request merged**